alarm/utils.py

```diff
@@ -0,0 +1,117 @@
+from alarm.models import Coin, Candle
+import websocket
+import json
+import ssl
+
+
+# Called whenever a message is received from the WebSocket server
+def feedback_message_from_websocket(coin_abbreviation, current_candle_high_price, threshold,
```

Comment:
From naming, we can't understand why the function is needed. If function needs a comment then the naming can be improved.

Reply:
I changed this naming to "extract_data_from_socket". What do you think?